package.json

```diff
@@ -23,16 +23,34 @@
     "url": "https://github.com/dojo/cli-create-theme.git"
   },
   "scripts": {
-    "test": "grunt test"
+    "test": "grunt test",
+    "build": "./node_modules/.bin/grunt release --pre-release-tag=rc --dry-run --skip-checks --initial --force"
```

Comment:
not sure we have this anywhere else.

Reply:
Removed, thought it could be useful for us while developing though